services/web-server/src/resolvers/WorkerManager.js

```diff
@@ -1,4 +1,15 @@
+const { splitWorkerPoolId } = require('taskcluster-worker-manager/src/util');
+
 module.exports = {
+  WorkerManagerWorkerPoolSummary: {
+    pendingTasks({ workerPoolId }, args, { loaders }) {
+      const { provisionerId, workerType } = splitWorkerPoolId(workerPoolId);
+      return loaders.pendingTasks.load({
```

Comment:
this is a request to the queue (`GET /pending/:provisionerId/:workerType`) - in the today's discussion in Slack turned out that in worker manager two parts of the `workerPoolId` have no meaning. My question is: does this request make sense here? And also: should there be a new analogous endpoint for worker manager?

Queue checks up on pulse queues for `pending` messages, but we probably won't be having such queues anymore?

Reply:
@djmitche or @imbstack ^ that is a question for you

Reply:
I think the request does make sense.  Queue is responsible for queues, so it's definitely the thing to ask about pending tasks.  And GraphQL is meant to integrate data from multiple backend services, so I think it makes sense to call both queue and worker-manager in this resolver.

For calling APIs that don't yet support `workerPoolId` or `taskQueueId`, we split it at the `/` and pass `provisionerId` and `workerType` separately, as you've don here.

I just wrote up an [RFC for this](https://github.com/taskcluster/taskcluster-rfcs/pull/145) that should hopefully give some background for that.